### PR TITLE
VC-4211 add empty accounts

### PR DESCRIPTION
**Pull request by alex-bodnar**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by IamStubborN — APPROVED** [review]
LGTM

**Comment by IamStubborN** [comment]
LGTM2

File: database/schema/11-accounts.sql [hunk 1]
```diff
@@ -0,0 +1,102 @@
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS vipcoin_chain_accounts_affiliates(
+    id               SERIAL PRIMARY KEY NOT NULL,                -- affiliates id
+    address          TEXT               NOT NULL,                -- affiliates address
+    affiliation_kind INT,                                        -- affiliates AffiliationKind
+    extras           JSONB                                       -- affiliates extras
+);
+
+CREATE TABLE IF NOT EXISTS vipcoin_chain_accounts_accounts (
```
Comment (DirusK) [inline database/schema/11-accounts.sql:9]: текущее состояние вроде просто как vipcoin_chain_accounts пишем

Comment (alex-bodnar) [reply database/schema/11-accounts.sql:9]: Мы с Андреем решили что первым идет имя модуля, а после имя таблицы.

Comment (DirusK) [reply database/schema/11-accounts.sql:9]: ок

**Comment by DirusK** [comment]
LGTM

**Pull request merged**